release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,551 @@
+xref:../scalability_and_performance/planning-your-environment-according-to-object-limits.adoc#planning-your-environment-according-to-object-limits[Cluster
+Limits] for {product-title} 4.2 is now available.
+
+Use the link:https://access.redhat.com/labs/ocplimitscalculator/[{product-title}
+Limit Calculator] to estimate cluster limits for your environment.
+
+[id="ocp-4-2-developer-experience"]
+=== Developer experience
+
+Builds on Windows nodes are not supported.
+
+
+[id="ocp-4-2-nodes"]
+=== Nodes
+
+[id="ocp-4-2-whitelisting-of-sysctls"]
+==== Whitelisting of sysctls configuration
+
+You can whitelist sysctls configurations. See
+xref:../nodes/containers/nodes-containers-sysctls.adoc#nodes-containers-sysctls[Using
+sysctls in containers] for more information.
+
+
+[id="ocp-4-2-multi-cluster"]
+=== Multi-cluster
+
+[id="ocp-4-2-resource-federation"]
+==== Resource federation (Technology Preview)
+
+[id="ocp-4-2-infrastructure"]
+=== Infrastructure
+
+[id="ocp-4-2-rhhi-next"]
+==== RHHI.Next
+
+RHHI.Next is a unified platform for container and VM workloads under common
+management control. Storage, compute, and networking is tightly integrated into
+{product-title} for a seamless customer experience. This includes:
+
+* {product-title}
+* OpenShift Container Storage
+* Container-native Virtualization
+* RHEL CoreOS
+* Industry Standard Server Hardware
+* Network Switches (optional)
+
+Benefits:
+
+* Get the infrastructure for your on-premise application development up and running quickly.
+* Hyperconverged infrastructure makes it easier to order, install, and manage.
+* Run virtualized workloads side-by-side with newer container workloads.
+* Consistent operational management with public cloud using {product-title}.
+
+[id="ocp-4-2-registry"]
+=== Registry
+
+[id="ocp-4-2-networking"]
+=== Networking
+
+[id="ocp-4-2-web-console"]
+=== Web console
+
+[id="ocp-4-2-custom-branding"]
+==== Console customization options
+
+You can customize the {product-title} web console to set a custom logo, links,
+notification banners, and command line downloads. This is especially helpful if you
+need to tailor the web console to meet specific corporate or government
+requirements.
+
+See Customizing the web console for more information.
+
+[id="ocp-4-2-new-api-explorer"]
+==== New API Explorer
+
+You can now easily search API resources in the *Explore API Resources* dashboard
+located at *Home* -> *Explore*.
+
+[id="ocp-4-2-prometheus-queries"]
+==== Prometheus queries
+
+You can now run Prometheus queries directly in the web console.
+
+[id="ocp-4-2-general-web-console-updates"]
+==== General web console updates
+
+* The dashboard is redesigned with more metrics.
+* *Catalog* is moved.
+* Status of projects is now moved to the *Workloads* menu.
+* OperatorHub is now located under the *Operators* menu.
+* There is now support for chargeback.
+* There is now support for native templates.
+
+[id="ocp-4-2-security"]
+=== Security
+
+FIPS mode is now supported.
```

Comment:
though this was tech preview in 4.2.. cc @knewcomerRH

Reply:
Not even tech preview. Please remove any reference to FIPS mode. Thanks!